### PR TITLE
Improved ta_stoch_bollinger strategy

### PR DESCRIPTION
**Pull request by jjdejong**
Improved trade responsiveness for limit cases where the closing and opening prices of a trend-change period fall on either side of a Bollinger limit. The sell/buy signals are now enabled if either the closing price or the opening price is outside the Bollinger bands. 

Also tweaked some default parameters that work better for me.

### CODE REVIEW

File: extensions/strategies/ta_stoch_bollinger/strategy.js [hunk 1]
```diff
@@ -40,50 +40,54 @@ module.exports = {
...
             s.period.stoch_K = inres.k[inres.k.length-1]
             var last_divergent = inres.k[inres.k.length-2] - inres.d[inres.d.length-2]
             var _switch = 0
-            var nextdivergent = (( divergent + last_divergent ) /2) + (divergent - last_divergent) 
-            if ((last_divergent <= 0 && (divergent > 0)) ) _switch = 1 // price rising 
-            if ((last_divergent >= 0 && (divergent < 0)) ) _switch = -1 // price falling
+            var nextdivergent = ((divergent + last_divergent)/2) + (divergent - last_divergent) 
+            if (last_divergent <= 0 && divergent > 0) {
+				_switch = 1 // price rising
+			}
```
Comment (DeviaVir) [inline extensions/strategies/ta_stoch_bollinger/strategy.js]: might need to format to match the rest of the file here

Comment (jjdejong) [reply extensions/strategies/ta_stoch_bollinger/strategy.js]: OK, applied VS Code beautifier and updated the switch color indicator

Comment (DeviaVir) [reply extensions/strategies/ta_stoch_bollinger/strategy.js]: Thanks!

**Review by DeviaVir — APPROVED** [review]
Thanks for contributing

**Pull request merged**